extensions/DynamicPageList/DPLSetup.php

```diff
@@ -1533,7 +1533,7 @@ public static function fixCategory($cat) {
     } 
 
 // reset everything; some categories may have been fixed, however via  fixcategory=
-    public static function endReset( &$parser, $text ) {
+    public static function endReset( $parser, $text ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$text". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)